src/Cache/CacheInterface.php

```diff
@@ -17,6 +17,8 @@
 /**
```

Comment:
> > adding readonly whenever possible
> 
> how are you sure all those are possible?

@kubawerlos , consider to use threads, otherwise hard to answer (it would look like random comments :D )

I added everywhere, and if it was failing (eg there is a `->setFoo()` method), i removed the annotation

Reply:
Can we test if all the classes could actually be readonly: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8277 ?

Reply:
fantastic!